### ISSUE TITLE
Remove the snapshot from FILEINFO

### ISSUE THREAD

**Issue by acdo2002** [CARTAvis/carta-backend-ICD-test#276]
Because we would like to monitor FILEINFO*.test.ts in more manual handling way, I decide to remove the snapshot from FILEINFO*.test.ts, then we may be clear to find the any change in header info.

**Comment by acdo2002** [comment]
This branch will based on the FILEINFO returned by [this](https://github.com/CARTAvis/carta-backend/pull/917) backend branch

**Comment by kswang1029** [comment]
using snapshot is still fine as long as the context is verified to be correct and the test framework can pick up differences between the snapshot and the generated file info from the backend.

**Comment by acdo2002** [comment]
Yes, but it may easily skip the difference, I think back to the previous code is more easily to monitor any change in the files' header. I just checkout to previous commit and copy them.
